src/components/MsgTransform.tsx

```diff
@@ -36,15 +35,15 @@ const MsgTransform = ({ text, textStyle }: Props) => {
         );
     };
 
-    const handleClickUser = (alias: string) => {
-        if (nickname === alias.replace('@', '')) {
-            changeScreen('MyIdeasScreen');
-        } else {
-            changeScreen('ProfileScreen', {
-                alias: alias.replace('@', ''),
-            });
-        }
-    };
+    // const handleClickUser = (alias: string) => {
```

Comment:
Remove this comment if this piece of code is unnecessary

Reply:
Done